.gitpod.yml

```diff
@@ -16,12 +16,7 @@ tasks:
     before: |
       redis-server &
     init: >
-      cp .env_sample .env &&
-      echo "APP_DOMAIN=\"$(gp url 3000 | cut -f 3 -d /)\"" >> .env &&
-      echo 'APP_PROTOCOL="https://"' >> .env &&
-      gem install solargraph &&
```

Comment:
I have no idea why this fails to install solar graph and/or foreman, which cause bin/startup to fail or never run. The bash commands look fine and copy pasting them into a terminal work fine. Maybe it's a formatting issue in the YAML file? I honestly don't know. Regardless, a script seemed to make more sense.